site/data/plugins.yml

```diff
@@ -0,0 +1,47 @@
+- name: Alert
```

Comment:
Maybe we should add Accordions, navbar, ...

Reply:
I'd say it's been done to fit perfectly vertically. Some from Bootstrap are missing as well. So I think there's no modification to do here. Thoughts?

Reply:
No, I think we had a discussion about it. We can go as-is.